op-supervisor/supervisor/backend/backend.go

```diff
@@ -259,9 +259,9 @@ func (su *SupervisorBackend) CheckBlock(chainID *hexutil.U256, blockHash common.
 	}
 	// at this point we have the extent of the block, and we can check if it is safe by various criteria
 	for _, checker := range []db.SafetyChecker{
-		db.NewSafetyChecker(types.Unsafe, su.db),
-		db.NewSafetyChecker(types.Safe, su.db),
-		db.NewSafetyChecker(types.Finalized, su.db),
+		db.NewSafetyChecker(db.Unsafe, su.db),
+		db.NewSafetyChecker(db.Safe, su.db),
+		db.NewSafetyChecker(db.Finalized, su.db),
```

Comment:
using `db` instead of `types` surprised me here but it's not an issue because this whole section gets removed soon.

Reply:
Yea, instead of using `types` I used the `db` constants that the method was accepting, to avoid having to pick local or cross, when it's really more about the high-level unsafe/safe/finality difference.